core/src/main/resources/hudson/model/Computer/sidepanel_es.properties

```diff
@@ -22,7 +22,7 @@
 
 Back\ to\ List=Volver
 Status=Estado
-Delete\ Slave=Borrar esclavo
+Delete\ Slave=Borrar agente
```

Comment:
This is now "Delete Agent"  - you changed the key in `core/src/main/resources/hudson/model/Computer/sidepanel.jelly`